sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_dsl_pipeline.py

```diff
@@ -1711,6 +1712,124 @@ def parallel_in_pipeline(job_data_path):
         assert_job_input_output_types(pipeline_job)
         assert pipeline_job.settings.default_compute == "cpu-cluster"
 
+    def test_parallel_run_function_run_settings_bind_to_liter_input(self, client: MLClient):
+        # command job with dict distribution
+        environment = "AzureML-sklearn-1.0-ubuntu20.04-py38-cpu:33"
```

Comment:
maybe you can extract the shared part comparing to `test_parallel_run_function` as a function like `build_pipeline_with_parallel_run`

Reply:
fixed.